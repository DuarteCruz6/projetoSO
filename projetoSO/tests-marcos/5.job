WRITE [(z8,83)(y4,76)(a5,31)(y3,10)]
READ [y3]
WRITE [(h8,70)(v3,4)(j4,13)(p8,99)]
WRITE [(g5,89)]
WRITE [(l9,88)(f0,12)(k3,62)(r7,88)]
WRITE [(m0,87)(s4,11)]
WRITE [(z4,86)(l7,18)(c2,92)(f2,22)(j5,5)]
DELETE [g5,h8,l9]
DELETE [y3,z8]
DELETE [a5,j5,j4]
DELETE [m0,p8,l7,v3,f0]
WRITE [(j1,34)(d6,28)]
DELETE [k3,d6,r7,c2,f2]
WRITE [(p1,60)(x2,25)(d7,40)]
WRITE [(j2,73)(h0,47)]
