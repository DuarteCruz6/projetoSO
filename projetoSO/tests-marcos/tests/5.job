WRITE [(t4,69)(k9,24)(g5,76)(h1,73)]
WRITE [(o4,73)(b9,0)(t0,20)]
BACKUP
WRITE [(z8,43)(r0,83)(g8,36)]
BACKUP
WRITE [(q6,39)(z0,94)(k6,38)(x9,3)(e1,53)]
WAIT 2000
WRITE [(v6,92)(v2,64)(x6,37)(j2,38)(m4,83)]
WRITE [(g1,46)]
WAIT 2000
DELETE [g1,m4,j2,x9]
WRITE [(n0,70)(e3,23)(p6,87)(u2,35)(t5,19)]
WRITE [(a6,95)(u5,65)(i3,23)(x3,76)]
WAIT 1000
READ [x3]
WRITE [(v9,78)]
WRITE [(b2,51)(f4,22)(y7,65)(h4,93)(b1,32)]
READ [b1]
WRITE [(b6,0)(w5,87)(w5,89)(h2,29)(c8,16)]
WRITE [(w4,50)(p2,23)(y6,91)]
DELETE [t5,b2,q6]
WRITE [(p3,88)]
WAIT 3000
DELETE [p6]
WRITE [(n5,61)(t2,22)(h5,3)]
WRITE [(o2,79)(p6,33)(i1,79)(z2,19)(k4,45)]
WRITE [(a1,28)(d6,0)(v7,77)(a3,65)(r4,51)]
READ [o2]
