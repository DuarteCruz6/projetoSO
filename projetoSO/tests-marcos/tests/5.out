[(x3,76)]
[(b1,32)]
[(o2,79)]
